docs/build-process/worker-build-process.md

```diff
@@ -0,0 +1,90 @@
+* If you are creating a worker from scratch, we recommend that you use the same SDK version as other workers in your project.
+
+**Notes:** 
+* You can mix SDK versions in an FPL project - in other words, you can build different workers within the same project using different SDK versions. This can be useful when transitioning a project to a newer SDK version - for example, you may opt to upgrade the SDK version of one worker at a time instead of upgrading the full project all at once.
+* Since there is no clearly defined SDK version for an FPL project, we recommend that you manually keep track of the SDK version(s) you are using to avoid forgetting that information.
+
+### Download the relevant worker SDK libraries
+Worker SDK libraries provide classes and functionalities for connecting with and participating in a SpatialOS simulation. They also provide functionalities for creating and manipulating snapshots.
+
+The choice of worker SDK library depends on the programming language and the target build platform of your worker. See the respective documentation for a full list of available worker SDK libraries:
+* C++: [Setting up a C++ worker](https://docs.improbable.io/reference/latest/cppsdk/setting-up)
+* C#: (Under construction)
+* Jave: (Under construction)
+* C: [Setting up a worker using the C API](https://docs.improbable.io/reference/latest/capi/setting-up#obtaining-the-sdk)
+
+You can use the `spatial package get` command to download zips that contain the worker SDK libraries of your choice. Below is an example of how to download the C# worker SDK library:
+
+```
+spatial package get --force --unzip worker_sdk csharp <my-sdk-version> <my-download-directory>
+```
+
+The following command flags are available:
+* `--force`: Overwrites the file to be downloaded if it already exists.
+* `--unzip`: Unzips the downloaded package. Creates specified directory if it does not exist.
+
+**Migration advice:** You can determine the worker SDK libraries that your existing worker depends on by inspecting the [`spatialos_worker_packages.json`](https://docs.improbable.io/reference/latest/shared/reference/file-formats/spatial-worker-packages) file in the directory of your [worker build configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-build). In the CPL, the `spatialos_worker_packages.json` specifies the worker SDK libraries that the spatial CLI would automatically download for you while building a worker.
+
+### Download the standard schema library
+The [standard schema library](https://docs.improbable.io/reference/latest/shared/schema/standard-schema-library) is a collection of `.schema` files that contain well-known components. Some standard schema library components are mandatory for all SpatialOS entities.
```

Comment:
Can we delete "well-known"? Should have caught this earlier. I don't think it's a very common term outside the company; could we go with something like "standard SpatialOS components" instead?

Reply:
How about "common components"? I don't like saying "the standard lib contains standard components"

Reply:
"Common components" sounds fine to me!